stat/distmv/statdist.go

```diff
@@ -207,6 +207,76 @@ func (KullbackLeibler) DistUniform(l, r *Uniform) float64 {
 	return logPx - logQx
 }
 
+// Renyi is a type for computing the Rényi divergence of order α from l to r.
+// The dimensions of the input distributions must match or the function will panic.
+//
+// The Rényi divergence with α > 0, α ≠ 1 is defined as
+//  D_α(l || r) = 1/(α-1) log(\int_x l(x)^α r(x)^(1-α)dx)
+// The Rényi divergence has special forms for α = 0 and α = 1. This type does
+// not implement α = ∞. For α = 0,
+//  D_0(l || r) = -log \int_x r(x)1{p(x)>0} dx
+// that is, the negative log probability under r(x) that l(x) > 0.
+// When α = 1, the Rényi divergence is equal to the Kullback-Leibler divergence.
+// The Rényi divergence is also equal to half the Bhattacharyya distance when α = 0.5.
+//
+// The parameter α must be in 0 ≤ α < ∞ or the distance functions will panic.
+type Renyi struct {
+	Alpha float64
+}
+
+// DistNormal returns the Rényi divergence between normal distributions l and r.
+// The dimensions of the input distributions must match or DistNormal will panic.
+//
+// For two normal distributions, the Rényi divergence is computed as
+//  Σ_α = (1-α) Σ_l + αΣ_r
+//  D_α(l||r) = α/2 * (μ_l - μ_r)'*Σ_α^-1*(μ_l - μ_r) + 1/(2(α-1))*[ln(|Σ_λ|/(|Σ_l|^(1-α)+|Σ_r|^α)]
```

Comment:
There is a parenthesis missing for `ln` in the last term, Σ_λ should be Σ_α, and + should be *. Btw, with [15] you cite my former colleague from my faculty :-)